en_us/shared/building_and_running_chapters/building_course/custom_courses.rst

```diff
@@ -75,9 +75,11 @@ enrolled in the course.
 #. Enter the CCX coach's username or email address and select **Add CCX
    Coach**.
 
-When the CCX coach next logs into the LMS, the **CCX Coach** tab will be
-visible. The CCX coach dashboard is not accessible by course team members with
-other roles.
+The user will be added as a coach, and if they're not enrolled in the course
+yet, they will automatically be enrolled (so they can access course content).
+If they're enrolled, they will get an email to let them know. When the CCX
```

Comment:
Hm good question. Will double check. And actually, I should clarify -- what you described is what I meant, but it's not clear from my language. Thanks.